nupic/encoders/category.capnp

```diff
@@ -0,0 +1,19 @@
+@0xb639caa166140b0f;
+
+using import "scalar.capnp".ScalarEncoderProto;
```

Comment:
I think using the absolute path might be better in nupic for consistency? What do you think?